### PR TITLE
Migrate casks from homebrew/cask-drivers.

### PR DESCRIPTION
**Pull request by reitermarkus**
Merge with https://github.com/Homebrew/homebrew-cask-drivers/pull/3421.

### CODE REVIEW

File: Casks/ftdi-vcp-driver.rb [hunk 1]
```diff
@@ -60,10 +63,19 @@
           /dev/tty.usbserial-XXXXXXXX
 
         where XXXXXXXX is a random ID, based on the serial number.
+
+        To uninstall the driver, you must use the GUI to move the
+        application from #{staged_path}/#{token} to Trash, otherwise
+        the system extension will not be removed.
```
Comment (reitermarkus) [inline Casks/ftdi-vcp-driver.rb:69]: Does this work? I tried this before but the extension is still in `/Library/SystemExtensions`. Unless that's a different one?

File: Casks/ftdi-vcp-driver.rb [hunk 2]
```diff
@@ -50,6 +49,10 @@
     #             args:       ["uninstall", "-", "com.ftdi.vcp.dext"],
     #           }
 
+    uninstall delete: [
+      "~/Library/Containers/com.ftdi.vcp.dext",
+      "~Library/Application Scripts/com.ftdi.vcp.dext",
```
Comment (reitermarkus) [inline Casks/ftdi-vcp-driver.rb:54]: Typo here.

File: Casks/ftdi-vcp-driver.rb (hunk 1 above)
Comment (razvanazamfirei) [reply Casks/ftdi-vcp-driver.rb:69]: I've been able to unload it as long as the uninstall call is coming from the app that installed the extension. If it gets orphaned (e.g. by `rm`-ing the app), I've only been able to uninstall it using `systemextensionsctl` and turning off SIP. 

We have similar issues with our uninstall process for `santa` and `avast-security`.

In theory, something like this should work more broadly, but I haven't tested it enough: 
```osascript
osascript \
    -e 'tell application "Finder"' \
    -e activate \
    -e 'move application file "FTDIUSBSerialDextInstaller" of folder "Applications" of startup disk to trash' \
    -e 'end tell'
```

File: Casks/ftdi-vcp-driver.rb (hunk 2 above)
Comment (razvanazamfirei) [reply Casks/ftdi-vcp-driver.rb]: Fixed. :)

File: Casks/ftdi-vcp-driver.rb (hunk 1 above)
Comment (razvanazamfirei) [reply Casks/ftdi-vcp-driver.rb:69]:
```shell
❯ systemextensionsctl list
2 extension(s)
--- com.apple.system_extension.driver_extension
enabled	active	teamID	bundleID (version)	name	[state]
*	*	658CPPCMJJ	com.ftdi.vcp.dext (1.1/010)	NullDriver	[activated enabled]
--- com.apple.system_extension.endpoint_security
enabled	active	teamID	bundleID (version)	name	[state]
*	*	irrelevant	com.irrelevant(irrelevant)	irrelevant	[activated enabled]
❯ sudo osascript \
          -e 'tell application "Finder"' \
          -e activate \
          -e 'move application file "FTDIUSBSerialDextInstaller" of folder "Applications" of startup disk to trash' \
          -e 'end tell'
Password:
application file FTDIUSBSerialDextInstaller.app of item .Trash of folder johnqpublic of folder Users of startup disk
❯ systemextensionsctl list
2 extension(s)
--- com.apple.system_extension.driver_extension
enabled	active	teamID	bundleID (version)	name	[state]
		658CPPCMJJ	com.ftdi.vcp.dext (1.1/010)	NullDriver	[terminating for uninstall but still running]
--- com.apple.system_extension.endpoint_security
enabled	active	teamID	bundleID (version)	name	[state]
*	*	irrelevant	com.irrelevant(irrelevant)	irrelevant	[activated enabled]

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**